plugins/aja/aja-ui-props.hpp

```diff
@@ -2,8 +2,8 @@
 
 #include <obs-module.h>
 
-static const char *kProgramOutputID = "aja_output";
-static const char *kPreviewOutputID = "aja_preview_output";
+#define AJA_OUTPUT_ID "aja_output"
+#define AJA_PREVIEW_OUTPUT_ID "aja_preview_output"
```

Comment:
Should I let them as macros, or should I turn them back to `static const char*` and add `[[maybe_unused]]` attribute to them ?

Reply:
I decided to use [[maybe_unused]].